pkg/client/keycloak/config.go

```diff
@@ -50,6 +50,18 @@ type KeycloakRealmConfig struct {
 	APIEndpointURI   string `json:"api_endpoint_uri"`
 }
 
+func (kc *KeycloakConfig) GetSSOProviderRealm() *KeycloakRealmConfig {
+	provider := kc.SelectSSOProvider
+	switch provider {
+	case MAS_SSO:
+		return kc.KafkaRealm
+	case REDHAT_SSO:
+		return kc.RedhatSSORealm
+	default:
+		return kc.KafkaRealm
+	}
+}
+
```

Comment:
By convention, Getters in Go should not have the `Get` prefix (see https://go.dev/doc/effective_go#Getters)
```suggestion
func (kc *KeycloakConfig) SSOProviderRealm() *KeycloakRealmConfig {
	provider := kc.SelectSSOProvider
	switch provider {
	case MAS_SSO:
		return kc.KafkaRealm
	case REDHAT_SSO:
		return kc.RedhatSSORealm
	default:
		return kc.KafkaRealm
	}
}

```